.github/workflows/daily-testgrid.yaml

```diff
@@ -0,0 +1,43 @@
+name: testGrid-daily
+on:
+  schedule:
+  - cron: "0 0 * * 1-5"
+
+defaults:
+  run:
+    working-directory: ./testgrid/tgrun
+
+jobs:
+  testgrid-init:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: kubeconfig
+      env:
+        KUBE_CONFIG_DATA: ${{ secrets.TESTGRID_KUBE_CONFIG_DATA }}
+      run: |
+        echo "$KUBE_CONFIG_DATA" | base64 --decode > /tmp/config
+
+    - name: setup-go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.15.1
+
+    - name: tgrun-build
+      run: make build
+
+    - name: testgrid-cleanup
+      env:
+        KUBECONFIG: "/tmp/config"
+      run:
+        kubectl delete vmi --all
+
+    - name: testgrid-queue
+      env:
+        KUBECONFIG: "/tmp/config"
+        TESTGRID_API: "https://api.testgrid.staging.kurl.sh/"
+      
+      run: |
+        ./bin/tgrun queue --ref $(date "+%s") --testgrid-api $TESTGRID_API
```

Comment:
What if we used the last 7 of the sha here?

Reply:
Sounds good, the time/date reference is carried by other metadata. Changing in the next commit.